modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm

```diff
@@ -12,7 +12,7 @@
 	worn_icon_state = "secmed_labcoat_blue"
 
 /obj/item/clothing/suit/hazardvest/security_medic
-	name = "security medic's vest"
+	name = "security medic vest"
 	desc = "A lightweight vest worn by the Security Medic."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
```

Comment:
While you're already here, would you miiiind fixing the emissives on this vest? It already has the emissive overlay set up and just needs an icon in the mob files: `"[icon_state]-emissive"`
It's easy enough to do: grab an already existing emissive sprite's opaque white, copy our icon state, remove all but the reflective material, and replace all colors of the reflective material with the opaque white. You're left with an opaque-white overlay to put atop the reflective material.

Sadly you'll need one for secmed_vest_blue too, for emissive code isn't... the best... but it'll be a copy so you can just dupe one and rename it

Reply:
Yeah i'll do it in the morning after I get back from work, along with addressing all your other reviews.